### PR TITLE
[libmupdf] Update to 1.16.1-epub-prerelease and build with msbuild in Windows

### PR DESCRIPTION
**Pull request by JackBoosY**
The existing CMakeLists.txt is missing many important build processes, converted to build using msbuild / make.

Related: #9416.

### CODE REVIEW

File: ports/libmupdf/portfile.cmake [hunk 1]
```diff
@@ -1,28 +1,66 @@
...
+    endif()
+    
+    # Bakup libmupdf project file to include gdcmjpeg include path
+    if (NOT EXISTS ${SOURCE_PATH}/platform/win32/libmupdf.vcxproj.bak)
+        configure_file(${SOURCE_PATH}/platform/win32/libmupdf.vcxproj ${SOURCE_PATH}/platform/win32/libmupdf.vcxproj.bak COPYONLY)
+        file(REMOVE ${SOURCE_PATH}/platform/win32/libmupdf.vcxproj)
+    endif()
+    configure_file(${SOURCE_PATH}/platform/win32/libmupdf.vcxproj.bak ${SOURCE_PATH}/platform/win32/libmupdf.vcxproj COPYONLY)
+    # Add include/gdcmjpeg/8 to include path
+    file(READ ${SOURCE_PATH}/platform/win32/libmupdf.vcxproj LIBMUPDF_CFG)
```
Comment (JackBoosY) [inline ports/libmupdf/portfile.cmake]: This cannot be solved using `/p:AdditionalIncludeDirectories="${CURRENT_INSTALLED_DIR}/include/gdcmjpeg/8"`

**Comment by JackBoosY** [comment]
@mamadymadan Could you test it?
Thanks.

**Comment by BillyONeal** [comment]
I'm closing this because @JackBoosY indicated out of band that he wasn't continuing work on it right now and this has been in 'requires author response' status for more than a month; Jack, please feel free to reopen if you want to continue.

Thanks for your contribution!

**Comment by JackBoosY** [comment]
In favor of #19714.

**Pull request closed without merging**